### PR TITLE
helpers.py: fixed pgcal settings according to wbc and loads the correct i2c addresses

### PR DESCRIPTION
**Pull request by michareichmann**

### CODE REVIEW

File: python/pxar_helpers.py [hunk 1]
```diff
@@ -65,6 +64,13 @@ def __init__(self, f):
                     parts = shlex.split(line)
                     if len(parts) == 2:
                         self.config[parts[0].lower()] = parts[1]
+                    elif len(parts) == 4:
+                        parts = [parts[0],' '.join(parts[1:])]
+                        print parts
```
Comment (simonspa) [inline python/pxar_helpers.py]: The print comments could be removed. If it's working we don't need garbage printout, if it's not working we need a fix...

File: python/pxar_helpers.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python2
```
Comment (simonspa) [inline python/pxar_helpers.py]: is there a reason for removing the shebang?

File: python/pxar_helpers.py [hunk 3]
```diff
@@ -163,12 +145,51 @@ def PxarStartup(directory, verbosity):
...
+    if int(config.get("nTbms")) == 0:
+        pg_setup = (
+            ("PG_RESR",25),
+            ("PG_CAL",pgcal),
+            ("PG_TRG",16),
+            ("PG_TOK",0))
+    else:
+        pg_setup = (
+            ("PG_RESR",15),
+            ("PG_CAL",106),
```
Comment (simonspa) [inline python/pxar_helpers.py]: this should probably also read `pgcal`

File: python/pxar_helpers.py [hunk 4]
```diff
@@ -163,12 +145,51 @@ def PxarStartup(directory, verbosity):
...
+            ("PG_TRG",0))
+
+       # Start an API instance from the core pxar library
```
Comment (simonspa) [inline python/pxar_helpers.py]: comment indentation to be fixed

File: python/pxar_helpers.py [hunk 5]
```diff
@@ -163,12 +145,51 @@ def PxarStartup(directory, verbosity):
     rocDacs = []
     rocPixels = list()
     rocI2C = []
-    print config.get("nrocs")
-    for roc in xrange(int(config.get("nrocs"))):
-        dacconfig = PxarParametersFile('%s%s_C%i.dat'%(os.path.join(directory,""),config.get("dacParameters"),roc))
+    config_nrocs = config.get("nrocs")
+    config_nrocs  = config_nrocs.split()
```
Comment (simonspa) [inline python/pxar_helpers.py]: this can be done in one line:

```
config_nrocs = config.get("nrocs").split()
```

**Comment by simonspa** [comment]
See inline comments, everything else is fine. It is worth mentioning that this also correctly sets the delay between CAL and TRG according to the WBC read from the config.

File: python/pxar_helpers.py (hunk 2 above)
Comment (michareichmann) [inline python/pxar_helpers.py]: no, just forgot to copy...^^

File: python/pxar_helpers.py [hunk 6]
```diff
@@ -65,6 +65,12 @@ def __init__(self, f):
                     parts = shlex.split(line)
                     if len(parts) == 2:
                         self.config[parts[0].lower()] = parts[1]
+                    elif len(parts) == 4:
+                        parts = [parts[0],' '.join(parts[1:])]
+                        if len(parts) == 2:
+                            self.config[parts[0].lower()] = parts[1]
+                            print parts[0].lower(), parts[1]
```
Comment (simonspa) [inline python/pxar_helpers.py]: `print` to be removed

**Pull request merged**